core/remoteviewserver.cpp

```diff
@@ -54,7 +54,9 @@ RemoteViewServer::RemoteViewServer(const QString &name, QObject *parent)
                                                     name), this, "clientConnectedChanged");
 
     m_updateTimer->setSingleShot(true);
-    m_updateTimer->setInterval(10);
+    // Qml render @ ~60fps which is approximatively 16.66ms, not interesting to throttle under that value
+    // because we would probably resend the same frame again.
```

Comment:
It was in the expectation that we would force request it, in this case unless for Qt 3D scene, we would not have a changed result frame, will change the comment.
I will add a test for that as well.